src/org/parosproxy/paros/network/ConnectionParam.java

```diff
@@ -223,6 +225,13 @@ protected void parse() {
         }
         
         try {
+            this.httpStateEnabled = getConfig().getBoolean(HTTP_STATE_ENABLED, true);

```

Comment:
Changed it. See https://github.com/zaproxy/zaproxy/pull/3422/commits/f823b8f4af36c52cb8080a6157afea4611d3a9f2